__tests__/ExpensiMark-Markdown-test.js

```diff
@@ -452,3 +452,35 @@ test('map real message with quotes', () => {
     const resultString = '\n> hi\n\n\n> hi\n\n';
     expect(parser.htmlToMarkdown(testString)).toBe(resultString);
 });
+
+test('Test heading1 markdown replacement', () => {
```

Comment:
I think you should split these tests up. For example this one is testing 

- normal `# Heading` case 
- cases with multiple spaces (`#   Heading`) 
- cases with none (`#Heading`). 

You can split this into 3 small and simple tests that'll be easier to read. Also this way there's no need for the long message in the test string that explains everything 😅

Reply:
yeah I like your suggestion 👍